source/openapi-admin-v3.yaml

```diff
@@ -3681,6 +3726,14 @@ components:
       required: true
       schema:
         type: string
+    DeploymentId:
```

Comment:
I'm not seeing this deploymentId in the draft endpoint documentation. Do we need to add this as a param around ln 2734?

Reply:
That endpoint uses `DraftId`, not `DeploymentId`. We use `DeploymentId` in the "Get a Deployment" and "Redeploy a Deployment" endpoints.

Reply:
Ah, I missed "current deployment" in the description. Was thinking you could pass both and get a diff. That'll show me for scanning too quickly!